### PR TITLE
Properly introduce IPLD

### PR DESCRIPTION
**Pull request by rvagg**
Expansion of content in #953.

@har00ga please steal, repurpose, incorporate this content as you see fit. Ping me if you have more questions on this or think a particular are could be expanded on.

### CODE REVIEW

File: src/systems/filecoin_nodes/repository/ipldstore/_index.md [hunk 1]
```diff
@@ -1,29 +1,35 @@
...
+
+Filecoin uses the **DAG-CBOR** codec for the serialization and deserialization of its data structures and interacts with that data using the IPLD Data Model and various tools that are built for the IPLD Data Model. Filecoin makes use of IPLD [Selectors]([selectors](https://github.com/ipld/specs/blob/master/selectors/selectors.md)) to address and retrieve subsets of linked data graphs. IPLD [Paths](https://github.com/ipld/specs/blob/master/data-model-layer/paths.md) are also used to address specific nodes within a linked data structure.
```
Comment (rvagg) [inline src/systems/filecoin_nodes/repository/ipldstore/_index.md:16]: The inclusion of "Paths" here is a bit pre-emptive. I don't know whether they are used anywhere in Filecoin at the moment but @ribasushi is currently working on something for the Lotus command line that will likely use Paths to address content within CAR files.

File: src/systems/filecoin_nodes/repository/ipldstore/_index.md [hunk 2]
```diff
@@ -1,29 +1,35 @@
...
+
+IPLD provides a consistent and coherent abstraction above data that allows Filecoin to build and interact with complex, multi-block data structures, such as its HAMT and Sharray <!-- TODO: links, correct names, other multi-block structures, AMT ? -->.
```
Comment (rvagg) [inline src/systems/filecoin_nodes/repository/ipldstore/_index.md:18]: Needs sanity checking, I'm doing this off an outdated and incomplete understanding of the data structures in Filecoin. They're arguably not strictly an IPLD concern the way they're implemented in Filecoin, but they still rest on top of IPLD and we're doing some work in the multi-block data structure space and have some specs that are _very close_ to Filecoin's HAMT and Sharray. See https://github.com/ipld/specs/tree/master/data-structures - it probably wouldn't take much content to describe precisely how the specs outlined there differ from the Filecoin ones. But maybe there's already good enough docs for them in this repo already.

**Comment by har00ga** [comment]
This is fantastic, thank you! 🙂

**Comment by har00ga** [comment]
Closing this as the info has been merged into the original PR -- thank you again! I may have some notes for expanding in the near future, but for now I think it suffices. Let me know your thoughts. :)

**Pull request closed without merging**